test/matchers/fake_flutter_matchers.dart

```diff
@@ -0,0 +1,438 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
For some of these files, are there cooresponding Java file in the flutter-intellij repo (or dart files in the flutter repo) that we should be keeping them up-to-date with? If so, we should call that out in comments in order to help future maintainers.

Reply:
Done.